content/Watsonx/WatsonxAI/110.md

```diff
@@ -0,0 +1,762 @@
+
+#### 1. Open the watsonx.ai console. If the sandbox project has not been created, click the Create a sandbox project button to create it.
+
+![](./images/110/Aspose.Words.84f85360-1bb4-4987-990e-fd6c65ab6365.003.png)
+
+#### 2. Select the Experiment with foundation models and build prompts tile.
+
+![](./images/110/Aspose.Words.84f85360-1bb4-4987-990e-fd6c65ab6365.004.png)
+
+#### 3. The Prompt Lab pages open. It should be in the Structured mode. If not, select the Structured tab.
+
+![](./images/110/Aspose.Words.84f85360-1bb4-4987-990e-fd6c65ab6365.005.png)
+
+#### 4. Click on the Model dropdown and select the flan-t5-xl-3b model (you may have to pick it out by clicking on View all foundation models).
+
+![](./images/110/Aspose.Words.84f85360-1bb4-4987-990e-fd6c65ab6365.006.png)
+
+#### 5. Enter the following text into the Instruction field:
+
+Classify the following complaint and determine which departments to route the complaint: Planning, Development, Service, Warehouse, L3, Packaging, and Marketing.
+
+#### 6. Enter the following text into the Input field in the Try section:
+
+Where are the 2 extra sets of sheets that are supposed to come with my order?![](./images/110/Aspose.Words.84f85360-1bb4-4987-990e-fd6c65ab6365.007.png)
+
+#### 7. Click Generate.
+
+![](./images/110/Aspose.Words.84f85360-1bb4-4987-990e-fd6c65ab6365.008.png)
+
+#### 8. The flan-t5-xl-3b model returns a completion of Marketing. This is not an unreasonable answer given that the flan model has no understanding of the business context. Since the complaint speaks of missing something extra , the flan model gave its best shot to generate the Marketing completion.
+
+Given the business use case and the background information (in Section 7.1), you want the model to respond with a completion of Warehouse and Packaging.
+
+Note: Try this out. Click the Generate button several more times. You should get the same answer (as expected). You will return to this point later.
+
+#### 9. Click on the Model parameters (![](./images/110/Aspose.Words.84f85360-1bb4-4987-990e-fd6c65ab6365.009.png) ) icon.
+
+![](./images/110/Aspose.Words.84f85360-1bb4-4987-990e-fd6c65ab6365.010.png)
+
+#### 10. On the slide-out panel, you see the following settings:
+
+```
+Decoding   Greedy
+Repetition penalty  1
+Stopping criteria   not set
+Min tokens  0
+Max tokens  20
+```
+
+#### 11. Change Decoding from Greedy to Sampling. More parameter settings are visible:
+
+```
+Temperature   0.7
+Top P   1
+Top K   50
+Random seed   not set
+```
+
+#### 12. Change the Temperature setting from 0.7 to 1.7 to allow more variations.
+
+#### 13. Click Generate.
+
+![](./images/110/Aspose.Words.84f85360-1bb4-4987-990e-fd6c65ab6365.011.png)
+
+#### 14. The flan-t5-xl-3b model returns with Promotion this time. This is again reasonable as this is a complaint about something missing so the problem can be with delivery. But this is not what you want, and Promotion is not a proper category.

```

Comment:
Probably note that the answer might vary with this higher temperature. It won't always be "Promotion"